libzecale/circuits/aggregator_circuit.tcc

```diff
@@ -148,6 +174,13 @@ libzeth::extended_proof<wppT, wsnarkT> aggregator_circuit<
         _pb.primary_input());
 }
 
+template<typename wppT, typename wsnarkT, typename nverifierT, size_t NumProofs>
+size_t aggregator_circuit<wppT, wsnarkT, nverifierT, NumProofs>::
+    num_primary_inputs() const
+{
+    return 1 + 1 + NumProofs * _num_inputs_per_nested_proof;
```

Comment:
Could you add a comment to justify the various `+1`'s here?

Reply:
added a comment.